commit cbae688d7df59938ccb4af534ec2ff75cbbe9221
Author: Koen Kooi <koen@dominion.thruhere.net>
Date:   Wed Jan 26 13:27:20 2011 +0100

    Revert "Set virtual size when configuring framebuffer"

    Virtual size too big, so revert it.

    This reverts commit 2653ef07883fbd1a5e7025f9300cf89b79ba429a.

	Upstream-Status: Pending

diff --git a/src/omapfb-crtc.c b/src/omapfb-crtc.c
index cbeff35..9655db2 100644
--- a/src/omapfb-crtc.c
+++ b/src/omapfb-crtc.c
@@ -98,8 +98,6 @@ OMAPFBCrtcCommitChangeMode (xf86CrtcPtr crtc)
 	v = ofb->state_info;
 	v.xres = mode->HDisplay;
 	v.yres = mode->VDisplay;
-	v.xres_virtual = crtc->scrn->virtualX;
-	v.yres_virtual = crtc->scrn->virtualY;
 	v.activate = FB_ACTIVATE_NOW;
 	v.pixclock = KHZ2PICOS(mode->Clock ? mode->Clock : 56000);
 	v.left_margin = mode->HTotal - mode->HSyncEnd;
